build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
     implementation 'org.springdoc:springdoc-openapi-starter-webmvc-ui:2.6.0'
     implementation 'org.springframework.boot:spring-boot-starter-webflux'
     implementation 'io.netty:netty-resolver-dns-native-macos::osx-aarch_64'
```

Comment:
지난번에 리뷰를 못 달았는데요..! 요 라이브러리가 어떤 역할인지 알 수 있을까요?

Reply:
macos에서의 netty 관련 에러가 발생했던걸로 기억합니다.
```

2024-10-28T10:45:47.115+09:00 ERROR 25137 --- [           main] i.n.r.d.DnsServerAddressStreamProviders  : Unable to load io.netty.resolver.dns.macos.MacOSDnsServerAddressStreamProvider, fallback to system defaults. This may result in incorrect DNS resolutions on MacOS. Check whether you have a dependency on 'io.netty:netty-resolver-dns-native-macos'. Use DEBUG level to see the full stack: java.lang.UnsatisfiedLinkError: failed to load the required native library
```

스프링부트를 실행할때 위와 같은 ERROR 로그가 출력됐었는데, 이에 대해 의존성을 설치하라는 안내문에 따라 설치하게 됐습니다.